radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/prometheus/PrometheusService.java

```diff
@@ -223,6 +234,26 @@ private void addBranchAndCommit(StringBuilder builder) {
     appendField(builder, "branch_and_commit", branchAndCommit);
   }
 
+  private void addCurrentFork(StringBuilder builder) {
+    appendField(builder, "current_fork_name", currentForkView.currentForkConfig().name());
+  }
+
+  private void addCandidateForkRemainingEpochs(StringBuilder builder) {
+    engineStatusService
+        .getCandidateForkRemainingEpochs()
+        .ifPresent(
```

Comment:
If not present, we should probably return 0 (or -1?) or something like that; so that the metric is reset, and it doesn't just think the value is missing (and hence use the last available value)